include/dts-functions.sh

```diff
@@ -1591,9 +1595,10 @@ send_dts_logs(){
 
     mkdir $log_dir
     cp ${DTS_LOG_FILE} $log_dir
+    cp ${DTS_VERBOSE_LOG_FILE} $log_dir
 
-    if [ -f ${ERR_LOG_FILE} ]; then
-      cp ${ERR_LOG_FILE} $log_dir
+    if [ -f ${ERR_LOG_FILE_REALPATH} ]; then
+      cp ${ERR_LOG_FILE_REALPATH} $log_dir
     fi
 
```

Comment:
`cp /tmp/a/b/c/file some/dir` would result in `some/dir/file` being created

Reply:
The final name will be the same as always e.g:
`QEMU_Standard_PC_(i440FX_+_PIIX,_1996)_1.16.3-3.fc41_754df0b9-bf8e-5beb-bc03-95d22bfae38c_2024_12_20_09_31_28_626870134.tar.gz`.

We are not using absolute path anywhere and even if we did then command would be e.g. `tar czf /tmp/log.tar.gz /tmp/log` which would create `log.tar.gz` file in tmp directory but:

* It's possible the command would fail because we would be trying to create file in folder we are trying to archive (recursion)
* files in archive itself would contain this path e.g. `tmp/log/...` (without leading `/`).

In other words it works ok